README.md

```diff
@@ -4,6 +4,9 @@
 
 > Using HTML, CSS, JavaScript, Npm and Webpack
 
+## Live View
+[Click here](
+https://shedrack-sunday.github.io/Leaderboard/dist/)
 
 ## Available Scripts
```

Comment:
- [ ] to make your readme file more professional kindly rename `Available Scripts` section in your readme file to Setup and add the  `npm install` as well with the other commands

Reply:
I've taken note of this, and it's fixed in my next commit.
Thanks for the suggestion.